android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java

```diff
@@ -18,4 +23,18 @@ public final class WalletConstants {
     // BraveWalletAddNetworksFragment
     public static final String ADD_NETWORK_FRAGMENT_ARG_ACTIVE_NETWORK = "activeNetwork";
     public static final String ADD_NETWORK_FRAGMENT_ARG_CHAIN_ID = "chainId";
+
+    public static List<String> BUY_SUPPORTED_NETWORKS = Arrays.asList(
+            BraveWalletConstants.MAINNET_CHAIN_ID, BraveWalletConstants.RINKEBY_CHAIN_ID,
+            BraveWalletConstants.ROPSTEN_CHAIN_ID, BraveWalletConstants.GOERLI_CHAIN_ID,
+            BraveWalletConstants.KOVAN_CHAIN_ID, BraveWalletConstants.LOCALHOST_CHAIN_ID,
```

Comment:
rinkeby, ropsten, goerly, kovan, localhost are not buy supported networks, on android we support only ethereum, desktop has Ramp integrated where on some other networks we can buy, but not on test ethereum networks.

Reply:
fixed: https://github.com/brave/brave-core/commit/c5dc6927b21d3bce27d2c348ad2d9cc88925d0ac